### PR TITLE
Pull Request for Version 0.1.0

### PR DESCRIPTION
**Pull request by MatiSalinas**
## Description
This pull request introduces version 0.1.0 of the project.

## Key Changes
- **Implemented Components:**
  - NavBar
  - Carousel
  - AboutUs
  - News
  - Faq
  - FormLab
  - MapGoogle
  - Contact

### CODE REVIEW

**Review by prietojulii — APPROVED** [review]

File: web/src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import './App.css'
+import AboutUs from './components/AboutUs'
+import CarouselImages from './components/CarouselImages'
+import Faq from './components/Faq'
+import FormLab from './components/FormLab'
+import NavBar from './components/NavBar'
+import News from './components/News'
+import MapGoogle from './components/MapGoogle'
+import Contact from './components/Contact'
```
Comment (GabrielEValenzuela) [inline web/src/App.jsx:9]: I think we can use the relative path `@` if is possible

File: web/src/components/AboutUs.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Sit quidem ullam nostrum, sapiente sunt error natus earum beatae facilis a corrupti dicta eius delectus quod eos voluptatem quibusdam voluptatum corporis!</p>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores nesciunt similique molestiae libero pariatur tempore quas provident, dignissimos beatae porro voluptatibus eligendi accusantium quaerat aliquid? Animi nostrum culpa sunt consequatur.</p>
+                    </div>
+
+            </div>
+        </div>
+
+    )
+}
+export default AboutUs
```
Comment (GabrielEValenzuela) [inline web/src/components/AboutUs.jsx:36]: Missing endline

File: web/src/components/CarouselImages.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+        &lt;
+      </button>
+      <button className="carousel__next" onClick={scrollNext}>
+        &gt;
+      </button>
+    </div>
+  )
+}
+export default CarouselImages
```
Comment (GabrielEValenzuela) [inline web/src/components/CarouselImages.jsx:37]: Missing endline

File: web/src/components/CarouselImages.jsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    if (emblaApi) emblaApi.scrollNext()
+  }, [emblaApi])
+
+  return (
+    <div className="carousel">
+      <div className="carousel__viewport" ref={emblaRef}>
+        <div className="carousel__container">
+          <div className="carousel__slide"><img src="https://img.freepik.com/premium-photo/computer-room_38810-865.jpg?w=1380" /></div>
+          <div className="carousel__slide"><img src="https://live.staticflickr.com/7285/16415815706_ce95427779_k.jpg" /></div>
+          <div className="carousel__slide"><img src="https://fcefyn.unc.edu.ar/media/images/WhatsApp_Image_2020-10-23_at.2e16d0ba.fill-1100x390-c100.jpg" /></div>
```
Comment (GabrielEValenzuela) [inline web/src/components/CarouselImages.jsx:24]: I think these images are only for test purpose right?

File: web/src/components/Contact.jsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+            <p>Instagram:https://www.instagram.com/fcefyn_unc/</p>
+            <p>youtube: https://www.youtube.com/channel/UCCBcN5DbyBA2C7iZBX4i9nw</p>
+            <p>linkedin: https://www.linkedin.com/school/fcefyn</p>
+            </div>
+        </div>
+
+)
+}
+
+export default Contact
```
Comment (GabrielEValenzuela) [inline web/src/components/Contact.jsx:24]: Endline

File: web/src/components/Faq.jsx [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+                    <h3>¿Cómo puedo sugerir una colaboración o un nuevo proyecto?</h3>
+                </div>
+                <motion.p variants={questionVariants} animate={isOpen.question4 ? "visible" : "hidden"}>Si tienes una idea para una colaboración o un nuevo proyecto, te invitamos a enviarnos una propuesta detallada por correo electrónico. Incluye una descripción del proyecto, tus objetivos y cualquier otro detalle relevante. Nuestro equipo revisará tu propuesta y se pondrá en contacto contigo para discutir posibles pasos a seguir.</motion.p>
+            </motion.div>
+            
+        </motion.div>
+    )
+}
+
+export default Faq
```
Comment (GabrielEValenzuela) [inline web/src/components/Faq.jsx:107]: Endline

File: web/src/components/FormLab.jsx
Comment (GabrielEValenzuela) [inline web/src/components/FormLab.jsx:1]: For form I think we can use https://www.react-hook-form.com/

**Review by GabrielEValenzuela — CHANGES_REQUESTED** [review]
Great work Mati, some improvements. Also, you made a fork of this repository?

I see that origin branch is `MatiSalinas:master`

Also, some commits can't be verified and the rules of the merge PR requires all commits verified

If you need we can schedule a meet and solve together 😃

File: web/src/components/CarouselImages.jsx (hunk 4 above)
Comment (MatiSalinas) [reply web/src/components/CarouselImages.jsx:24]: Yes, those are for test purposes, actual images will be stored in the public folder.

File: web/src/App.jsx (hunk 1 above)
Comment (MatiSalinas) [reply web/src/App.jsx]: Done.

File: web/src/components/FormLab.jsx
Comment (MatiSalinas) [reply web/src/components/FormLab.jsx:1]: Can we schedule a meeting, I have a few questions regarding this library.

**Comment by MatiSalinas** [comment]
Indeed, I made a fork of the repository, should I work directly on the main one?
In my fork, I have additional changes that allow the components to be responsive on mobile.

I made the corrections for the missing endlines and the relative path @. I also amended the unverified commits.

**Comment by prietojulii** [comment]
Close This PR.

**Pull request closed without merging**